src/App/App.tsx

```diff
@@ -47,15 +47,25 @@ export default function App() {
               <Route path={`${appRoutes.donate}/:id`} element={<Donate />} />
               <Route path={appRoutes.leaderboard} element={<Leaderboard />} />
               <Route
+                index={IS_AST}
                 path={`${appRoutes.register}/*`}
                 element={IS_AST ? <Launchpad /> : <Registration />}
               />
               <Route path={`${appRoutes.gift}/*`} element={<Gift />} />
-              <Route index element={<Marketplace />} />
+              <Route
+                index={!IS_AST}
+                path={appRoutes.marketplace}
+                element={<Marketplace />}
+              />
             </Route>
             <Route
               path="*"
-              element={<Navigate replace to={appRoutes.index} />}
+              element={
+                <Navigate
+                  replace
+                  to={IS_AST ? appRoutes.register : appRoutes.index}
```

Comment:
Error encountered on app startup.
Repro steps:
- open `.env`
- comment out `REACT_APP_APP_TYPE` or set value to something other than "AST"
- start webapp
- open browser console
- go to index route http://localhost:4200/
- **expected:** Marketplace page appears
- **actual:** blank page appears together with error in console like in image below:
![image](https://user-images.githubusercontent.com/19427053/231393674-977c2d80-ad7d-4729-91a2-39cfde0fa0e8.png)

Reply:
Resolved